topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -420,11 +603,8 @@ Let's first have a look at the quality of our reads!
 >    ```
 >
 > 2. Rename the file to `reads_2`
-> 3. **FastQC** {% icon tool %} with the reverse reads
->
->     FastQC has already been run on the forward reads
->
-> 4. **MultiQC** {% icon tool %} with the following parameters to aggregate the FastQC reports
+> 3. **FastQC** {% icon tool %} with the reverse reads>
```

Comment:
Looks like `>` at the end of the line is not needed.

Reply:
Nice catch!